components/centraldashboard/public/components/dashboard-view.js

```diff
@@ -119,20 +119,41 @@ export class DashboardView extends PolymerElement {
      * Observer for platformInfo property
      */
     _platformInfoChanged() {
-        if (this.platformInfo && this.platformInfo.providerName === 'gce') {
-            this.platformName = 'GCP';
-            const pieces = this.platformInfo.provider.split('/');
-            let gcpProject = '';
-            if (pieces.length >= 3) {
-                gcpProject = pieces[2];
-            }
-            this.platformLinks = GCP_LINKS.map((l) => {
-                return {
-                    text: l.text,
-                    link: l.link + gcpProject,
-                };
-            });
+        if (!this.platformInfo || this.platformInfo.providerName != 'gce') {
```

Comment:
The return statement needs to be uncommented if you were to keep this as-is. I used the positive condition in anticipation of adding additional links for other platforms (AWS, Azure, etc.) where we could first check platformInfo and then have another if block or a switch statement for the platform.

Reply:
Yes, I will actually revert it before merging since we'll probably have links relevant to different platforms.